src/options.js

```diff
@@ -41,19 +48,23 @@ async function saveOptions() {
 }
 
 async function restoreOptions() {
-    chrome.storage.sync.get(['metube', 'sites', 'sendOnClick'], function (data) {
+    chrome.storage.sync.get([
+        'metube',
+        'sites',
+        'clickBehavior',
+        'contextMenuClickBehavior'
+    ], function (data) {
         if (data.metube != undefined) {
             document.getElementById("metube").value = data.metube;
         }
 
         if (data.sites != undefined) {
             document.getElementById("additional").value = data.sites;
         }
-        // document.getElementById("send-on-click").checked = true;
-        if (data.sendOnClick) {
-            document.getElementById("send-on-click").checked = true;
-        }
-    });
+
+        document.getElementById(data.clickBehavior).checked = true;
+        document.getElementById(data.contextMenuClickBehavior).checked = true;
```

Comment:
you need to additional check here because `data.clickBehavior` can be `undefined`

<img width="722" alt="screenshot-2022-10-07-12-01-42-28d4b6fea0" src="https://user-images.githubusercontent.com/265634/194503932-4f1aae99-028e-4302-a42a-b8e99e0deeb6.png">

Also you can use another strategy and set default values if they not presented in `data` object

Reply:
data.clickBehavior should always be set when a user first saves the options as one radio box is always pre-checked as default for the user for both clickBehavior and contextMenuClickBehavior

I'm actually amazed how you even ran into this - the extension does not work at all if the options aren't saved at least once by the user, and if they are saved once by the user, then valid data.clickBehavior and data.contextMenuClickBehavior values will be set

Reply:
when user just install it this values not setted, just add check or set default values in the install action

Reply:
Done - added the variables to be set at extension install time as requested.
I also took your suggestion and added the check since it doesn't hurt anyway.